pkgs/development/python-modules/auditok_0_1_5/default.nix

```
@@ -0,0 +1,37 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, matplotlib
+, numpy
+, pyaudio
+, pydub
+, unittestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "auditok";
+  # The most recent version is 0.2.0, but the only dependent package is
+  # ffsubsync, which is pinned at 0.1.5.
+  version = "0.1.5";
```

Limiting new module to supported Python releases avoid dealing with legacy Python version. Also, the change is high that nobody will use it on Python 3.1.

```suggestion
  version = "0.1.5";
  format = "setuptools";

  disabled = pythonOlder "3.7";
```